### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by mekarpeles**
- Creates a persistant DB
- sets up TBBO on ol-dev1 (runs on port 31337)

Next step:
- switch/point thebestbookon.com --> 207.241.231.169:31337
- recover data (?) from OJF
- @cdrini, any way for us to load OSP into solr? (we have dumps mapping counts to OL work)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -7,14 +7,14 @@ docopt==0.6.2
 Flask==1.1.2
 Flask-Routing==0.0.21
 idna==2.10
-internetarchive==1.9.4
+internetarchive==2.1.0
```
Comment (cclauss) [inline requirements.txt:10]: 2.2.0 ?  https://pypi.org/project/internetarchive

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 venv/
```
Comment (cclauss) [inline .gitignore:1]:
```suggestion
*venv/
```
The Python venv docs actually recommend `.venv` so it is best to be ready for both.

File: requirements.txt (hunk 1 above)
Comment (mekarpeles) [reply requirements.txt:10]: We removed the dependencies and let pip install the right matches and these are the changes it proposed, so I'm going to go with them for now

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,3 @@
-venv/
+*venv/
 *.egg-info/
+*.log
```
Comment (jimchamp) [inline .gitignore:3]:
```suggestion
*.log

```
Newline at end of file.

**Review by jimchamp — APPROVED** [review]

**Pull request merged**